lib/containers/container_images.pm

```diff
@@ -52,10 +52,38 @@ sub build_container_image {
 
 # Build a sle container image using zypper_docker
 sub build_with_zypper_docker {
-}
+    my %args    = @_;
+    my $image   = $args{image};
+    my $runtime = $args{runtime};
+    my $derived_image = "zypper_docker_derived";
```

Comment:
```suggestion
    my $derived_image = "zypper_docker_derived_$image";
```

Reply:
clever trick, would save some hassle in some cases!